README.md

```diff
@@ -47,6 +46,15 @@ Then run the download script (make sure to change `<DESTINATION_FOLDER>`):
 python dataset_download/download_dataset.py --download_folder <DESTINATION_FOLDER> --checksum_check
 ```
 
+Option2: Using the standard download script with HuggingFace option
+You can also use the original download script with the HuggingFace option:
```

Comment:
This seems repetitive.

Please also change the text above to include ”Option 1”.